### ISSUE TITLE
WebMapServiceImageryProvider clobbers srs option

### ISSUE THREAD

**Issue by tamarmot** [AnalyticalGraphicsInc/cesium#6223]
If, theoretically, one were using a different SRS, such as one for the moon, and wanted to pass that through to a WMS via the WebMapServiceImageryProvider, there is currently no way to do so as this is hardcoded as follows:

// Use SRS or CRS based on the WMS version.
        if (parseFloat(resource.queryParameters.version) >= 1.3) {
            // Use CRS with 1.3.0 and going forward.
            // For GeographicTilingScheme, use CRS:84 vice EPSG:4326 to specify lon, lat (x, y) ordering for
            // bbox requests.
            parameters.crs = options.tilingScheme instanceof WebMercatorTilingScheme ? 'EPSG:3857' : 'CRS:84';
        } else {
            // SRS for WMS 1.1.0 or 1.1.1.
            parameters.srs = options.tilingScheme instanceof WebMercatorTilingScheme ? 'EPSG:3857' : 'EPSG:4326';
        }

It would be a simple thing to check for the srs parameter in the options, and use that instead.

**Comment by hpinkos** [comment]
Thanks @tamarmot! If you have a chance, we'd be happy to take another PR with this fix =)

**Comment by tamarmot** [comment]
Yep, I'll be looking into this soon now that I know the process!

**Comment by anne-gropler** [comment]
This is done, isn't it? The `Cesium.WebMapServiceImageryProvider.ConstructorOptions` contain `crs` and `srs`.
